### PR TITLE
kafka: Add producer message retries config values.

### PR DESCRIPTION
**Pull request by ldjebran**
# Description
Add producer configMap keys:
* request.required.acks
* message.send.max.retries
* retry.backoff.ms This is also used in some sibling projects https://github.com/RedHatInsights/playbook-dispatcher/blob/master/internal/common/kafka/kafka.go#L24
More info about the keys: https://camel.apache.org/components/3.20.x/kafka-component.html

added to configuration only this values:
```go 

	KafkaRequestRequiredAcks   int                       `json:"kafka_request_required_acks,omitempty"`
	KafkaMessageSendMaxRetries int                       `json:"kafka_message_send_max_retries,omitempty"`
	KafkaRetryBackoffMs        int                       `json:"kafka_retry_backoff_ms,omitempty"`

```
FIXES: THEEDGE-3000

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [X] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by omaciel — APPROVED** [review]
/lgtm

File: pkg/common/kafka/kafkaconfigmap.go [hunk 1]
```diff
@@ -26,8 +26,8 @@ func NewKafkaConfigMapService() KafkaConfigMapServiceInterface {
 	return &KafkaConfigMapService{}
 }
 
-// GetKafkaProducerConfigMap returns the correct kafka auth based on the environment and given config
-func (k *KafkaConfigMapService) GetKafkaProducerConfigMap() kafka.ConfigMap {
+// getKafkaCommonConfigMap returns the kafka configMap common to producer and consumer
```
Comment (mgold1234) [inline pkg/common/kafka/kafkaconfigmap.go:29]: can you add unit test for this function?

**Comment by mgold1234** [comment]
thank you @ldjebran for explanation why we don't need unit test for the GetKafkaCommonConfigMap function. approve

**Review by mgold1234 — APPROVED** [review]
/lgtm

**Pull request merged**